doc/contribute/bin_blobs.rst

```diff
@@ -0,0 +1,230 @@
+.. _bin-blobs:
+
+.. warning::
+
+   This page documents Zephyr's support for binary blobs. The tooling and
+   infrastructure for this purpose are still incomplete, and so this page may
+   refer to future additions.
+
+Binary Blobs
+************
+
+In the context of an operating system that supports multiple architectures and
+many different IC families, some functionality may be unavailable without the
+help of executable code distributed in binary form.  Binary blobs (or blobs for
+short) are files containing proprietary machine code or data in a binary format,
+e.g. without corresponding source code released under an OSI approved license.
+
+Zephyr supports downloading and using third-party binary blobs via its built-in
+mechanisms, with some important caveats, described in the following sections. It
+is important to note that all the information in this section applies only to
+`upstream (vanilla) Zephyr <https://github.com/zephyrproject-rtos/zephyr>`_.
+
+There are no limitations whatsoever (except perhaps license compatibility) in
+the support for binary blobs in forks or third-party distributions of Zephyr. In
+fact, Zephyr’s build system supports arbitrary use cases related to blobs. This
+includes linking against libraries, flashing images to targets, etc. Users are
+therefore free to create Zephyr-based downstream software which uses binary
+blobs if they cannot meet the requirements described in this page.
+
+Software license
+================
+
+Most binary blobs are distributed under proprietary licenses which vary wildly
+in nature and conditions.  Licenses will be approved individually by the
```

Comment:
The TSC to approve license of binary blobs that are not being added into zephyr directly? I look at the original document and this applies only to the porting layer, not to the blob itself. Can you please point me where this is coming from?

Reply:
Yes, you are right, there is no mention of license approval by the TSC at all in the original document. Should we then not check licenses for the blob itself at all, and let vendors just populate that freely?